kbase-extension/static/kbase/js/common/cellComponents/tabs/results/outputWidget.js

```diff
@@ -14,26 +28,57 @@ define(['bluebird', 'common/html', 'common/ui', 'util/kbaseApiUtil'], (
         tr = tag('tr'),
         th = tag('th'),
         td = tag('td'),
-        table = tag('table');
+        thead = tag('thead'),
+        tbody = tag('tbody'),
+        table = tag('table'),
+        a = tag('a'),
+        tablePageLength = 50;
 
     function OutputWidget() {
         let container, ui;
 
         /**
          *
-         * @param {Array} objectData an array of report object references
+         * @param {Array} objectData an array of object data. Each element is expected
+         *  to have these properties:
+         *  - type - the workspace object type
+         *  - name - the name of the object
+         *  - description - a description of the object
          */
         function renderOutput(objectData) {
             if (objectData.length === 0) {
-                return div('No objects created');
+                return {
+                    layout: div('No objects created')
+                };
             }
-            return table({ class: 'table table-bordered table-striped' }, [
-                tr([th('Created Object Name'), th('Type'), th('Description')]),
-                ...objectData.map((obj) => {
-                    const parsedType = APIUtil.parseWorkspaceType(obj.type);
-                    return tr([td(obj.name), td(parsedType.type), td(obj.description)]);
-                }),
+            const events = Events.make();
+            const layout = table({
+                class: 'table table-bordered',
+                dataElement: 'objects-table',
+            }, [
+                thead(
+                    tr([th('Created Object Name'), th('Type'), th('Description')]),
+                ),
+                tbody(
+                    [...objectData.map((obj) => {
+                        const parsedType = APIUtil.parseWorkspaceType(obj.type);
+                        const objLink = a({
+                            href: '#',
+                            style: 'cursor: pointer;',
+                            dataObjRef: obj.ref,
+                            id: events.addEvent({
+                                type: 'click',
+                                handler: () => {
+                                    Jupyter.narrative.addViewerCell(obj.wsInfo);
```

Comment:
This was the original behavior of the created objects list. It somehow passed inspection without that part...